src/ekat/ekat_pack.hpp

```diff
@@ -292,6 +292,58 @@ struct Pack {
     return *this;
   }
 
```

Comment:
I would add a short, general comment for update methods that has the `y = beta*y + alpha*x` pseudo code.